frontend/src/layout/Topcontent.js

```diff
@@ -68,6 +73,14 @@ export class Topcontent extends Component {
                         </span>
                     )}
                     {this.props.user.email}
+                    <NavLink to="/logout">
```

Comment:
@jay-khatri A much easier way of doing this is just using an `<a href='/logout'>Logout</a>`!

Reply:
@timgl got it; just replaced the Navlink+Button with the `a` tag. Let me know if you have any thoughts on the styling?

Reply:
Hm I think this would actually be better as a dropdown, so we can add more links like this in the future. So something like

```js
<Dropdown
  title={this.props.user.email}
>
   <a className='dropdown-item' href='/logout'>Logout</a>
</Dropdown>
```

What do you think?